src/main/java/org/testcontainers/containers/TarantoolCartridgeContainer.java

```diff
@@ -186,13 +190,44 @@ public TarantoolCartridgeContainer(String dockerFile, String buildImageName, Str
                 instancesFile, topologyConfigurationFile);
     }
 
-    private TarantoolCartridgeContainer(Future<String> image, String instancesFile, String topologyConfigurationFile) {
+    /**
+     *
+     * @param dockerFile                URL resource path to a Dockerfile which configures Cartridge
+     *                                  and other necessary services
+     * @param buildImageName            Specify a stable image name for the test container to prevent rebuilds
+     * @param instancesFile             URL resource path to instances.yml relative in the classpath
+     * @param buildArgs                 a map of arguments that will be passed to docker ARG commands on image build.
+     *                                  This values can be overridden by environment.
+     */
+    public TarantoolCartridgeContainer(String dockerFile, String buildImageName, String instancesFile,
```

Comment:
why do you need this ctor ? this class has too much of them already

Reply:
This is constructor using the we haven't file topologi.lua (topology get whith replicasets.yaml default)

Reply:
Did refactoring after change definition configuration strategy